src/learning/nnet.rs

```diff
@@ -23,7 +24,7 @@
 //! let criterion = BCECriterion::new(Regularization::L2(0.1));
 //!
 //! // We will just use the default stochastic gradient descent.
-//! let mut model = NeuralNet::new(layers, criterion, StochasticGD::default());
+//! let mut model = NeuralNet::mlp(layers, criterion, StochasticGD::default(), Sigmoid);
```

Comment:
Similar to comment on examples - as this is the module docs I think we should be super clear about what this line is doing.